tests/conftest.py

```diff
@@ -122,7 +123,7 @@ def run_and_parse(testdir, exit_code_exp=0, runpytest_args=None):
 
     junit_xml_doc = JunitXml(str(result_path))
 
-    return junit_xml_doc
+    return (junit_xml_doc, result)
```

Comment:
I updated `run_and_parse` to return the tuple.  This minimized the changes to the tests.

Reply:
Parenthesis are unnecessary in this context.